client/pages/AssessmentSection/DataTable/GenerateValues/method.ts

```diff
@@ -13,29 +7,29 @@ export enum Method {
 }
 
 // TODO: Update to support our data model
-
-export const Methods = {
-  isValid: (data: TableData, method: Method, fields: Array<GenerateValuesField>): boolean => {
-    // Clear table doesn't need any fields selected
-    if (method === Method.clearTable) {
-      return true
-    }
-    const fieldsSelected = fields.filter((field) => field.selected === true)
-    // method and fields must be selected
-    if (Objects.isEmpty(method) || Objects.isEmpty(fieldsSelected)) {
-      return false
-    }
-    // Annual change
-    // check that for all selected fields, past and future chage rates are not empty
-    if (method === Method.annualChange) {
-      return fieldsSelected.every((field) => {
-        const { annualChangeRates } = field
-        return !Objects.isEmpty(annualChangeRates.past) && !Objects.isEmpty(annualChangeRates.future)
-      })
-    }
-    // Linear or Repeat last require a minimum number of ODP
-    const noOdpsRequired = method === Method.linear ? 2 : 1
-    const odps: any = FraUtils.getOdps(data)
-    return odps.length >= noOdpsRequired
-  },
-}
+//
+// export const Methods = {
```

Comment:
why commented out?

Reply:
Not used, (todo) see: https://github.com/openforis/fra-platform/pull/1477/files#diff-7dcd6666582341b145eb073ab080acf381031134866fd08b83e9137d565ec837L96